docs/src/pages/guide/getting-started/installation.mdx

```diff
@@ -38,6 +38,13 @@ mkdir hello-mastra
 cd hello-mastra
 ```
 
+Initialize the project and install @mastra/core:
+
+```bash copy
+npm init
+npm i @mastra/core
+```
+
 Run the `mastra init` command:
```

Comment:
@rgarcia `mastra init` handles the setup for you. Is there something we could do better to make the user know the option exists?

Reply:
didn't work for me:

```
bun i -g mastra
mkdir hello-mastra
cd hello-mastra
mastra init
✖️ No package.json file found in the current directory
```

Reply:
Oh wow! Our CI isn't tagging latest on releases, so the default installed cli is OLD! :(
Going to fix that now and then it should work!

In the meantime if you wanted to `bun i -g mastra@alpha` it would unblock this.

Reply:
We tagged all our latest releases now so i believe this is resolved!